### ISSUE TITLE
Very low values of recall 

### ISSUE THREAD

**Issue by shivangibithel** [ExID-proj/VILBERT_tutorial#1]
I tried to run your code for the flickr30k test dataset, but it is giving very low final value of recall. 
Can you confirm if this is the case with you as well...
![image](https://user-images.githubusercontent.com/19774925/129276290-521c5c26-75e7-49e5-bea3-db4289207d0b.png)

**Comment by Souryadipstan** [comment]
Off topic,

Have you tried running the code for any other dataset than flickr30 dataset? I tried but it is not working, I think it is hard coded for flickr30 dataset.

**Comment by shivangibithel** [comment]
No. I switched to ViLT later for my own datasets. They way bounding box features are generated is not clear to me in this case...thus working for my own dataset was difficult here.

**Comment by Souryadipstan** [comment]
The bounding box features are generated by using FASTER RCNN model combined with mean pooling later on as mentioned in the ViLBERT paper I think. managed to do that somehow but the work is still in process. To get it working I had to change the code a little bit as the code would have worked only if the image file names were written as integers. I am yet to check the performance of the model.

This is mentioned in the ViLBERT paper:

_We use Faster R-CNN [31] (with ResNet-101 [11] backbone) pretrained on the Visual Genome dataset [16] (see [30] for details) to extract region features. We select regions where class detection probability exceeds a confidence threshold and keep between 10 to 36 high-scoring boxes. For each selected region i, vi is defined as the mean-pooled convolutional feature from that region._
